app/src/utils/templatetags/time_utils.py

```diff
@@ -15,3 +15,8 @@ def server_time():
 @register.filter
 def zoneformat(time, zonename):
     return format_in_zone(time, zonename, DEFAULT_TIME_FORMAT)
+
+
+@register.filter
+def isoformat(time):
+    return time.strftime("%Y-%m-%dT%H:%M:%SZ")
```

Comment:
In ISO standard the letter 'Z' at the end of format string declares UTC time only. I'd rather first convert input time to UTC or specify offset instead.

Reply:
JS can’t parse date if you don’t add this `Z` ...

Reply:
Nope, you can omit it, but then it'll be treated as local time: https://www.ecma-international.org/ecma-262/#sec-date.parse

Reply:
Is it ok right now?

Reply:
OK